src/Core/Grid/Definition/Factory/CreditSlipGridDefinitionFactory.php

```diff
@@ -0,0 +1,188 @@
+     */
+    public function __construct(HookDispatcherInterface $hookDispatcher, $dateFormat)
+    {
+        parent::__construct($hookDispatcher);
+        $this->dateFormat = $dateFormat;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getId()
+    {
+        return self::GRID_ID;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getName()
+    {
+        return $this->trans('Credit slips', [], 'Admin.Navigation.Menu');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getColumns()
+    {
+        return (new ColumnCollection())
+            ->add((new DataColumn('id_order_slip'))
+                ->setName($this->trans('ID', [], 'Admin.Global'))
+                ->setOptions([
+                    'field' => 'id_order_slip',
+                ])
+            )
+            ->add((new DataColumn('id_order'))
+                ->setName($this->trans('Order ID', [], 'Admin.Orderscustomers.Feature'))
+                ->setOptions([
+                    'field' => 'id_order',
+                ])
+            )
+            ->add((new DateTimeColumn('date_add'))
+                ->setName($this->trans('Date issued', [], 'Admin.Orderscustomers.Feature'))
+                ->setOptions([
+                    'field' => 'date_add',
+                    'format' => $this->dateFormat,
+                ])
+            )
+            ->add(
+                (new LinkColumn('pdf'))
+                    ->setName($this->trans('PDF', [], 'Admin.Global'))
+                    ->setOptions([
+                        'sortable' => false,
+                        'field' => 'link_value',
+                        'route' => 'admin_credit_slips_pdf',
+                        'route_param_name' => 'creditSlipId',
+                        'route_param_field' => 'id_order_slip',
+                        'icon' => 'insert_drive_file',
+                        'button_template' => 'outline',
+                        'color_template' => 'secondary',
+                    ])
+            )
+            ->add((new ActionColumn('actions')));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFilters()
+    {
+        return (new FilterCollection())
+            ->add((new Filter('id_credit_slip', TextType::class))
+                ->setTypeOptions([
+                    'required' => false,
+                    'attr' => [
+                        'placeholder' => $this->trans('Search ID', [], 'Admin.Actions'),
+                    ],
+                ])
+                ->setAssociatedColumn('id_order_slip')
+            )
+            ->add((new Filter('id_order', TextType::class))
+                ->setTypeOptions([
+                    'required' => false,
+                    'attr' => [
+                        'placeholder' => $this->trans('Search order ID', [], 'Admin.Orderscustomers.Feature'),
```

Comment:
In this case, I would follow the localization of the previous string and write _Admin.Actions_ as domains.